pom.xml

```diff
@@ -185,27 +207,17 @@
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-invoker-plugin</artifactId>
-              <version>3.2.2</version>
               <configuration>
-                <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
                 <cloneClean>true</cloneClean>
                 <pomIncludes>
                   <pomInclude>*/pom.xml</pomInclude>
                   <pomInclude>*/non-default-pom.xml</pomInclude>
                 </pomIncludes>
-                <preBuildHookScript>setup</preBuildHookScript>
-                <postBuildHookScript>verify</postBuildHookScript>
-                <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
-                <settingsFile>src/it/settings.xml</settingsFile>
                 <addTestClassPath>true</addTestClassPath>
                 <goals>
                   <goal>clean</goal>
                   <goal>install</goal>
                 </goals>
-                <properties>
-                  <!-- e.g. ensure that Java7 picks up TLSv1.2 when connecting with Central -->
-                  <https.protocols>${https.protocols}</https.protocols>
```

Comment:
This is needed because <javaVersion>7</javaVersion>.

Reply:
We don't even build on JDK7 neither on Github nor on ASF jenkins...

Reply:
Identical configuration is in parent, so is not needed because is inherited and of course is only needed when we want build by jdk 1.7